.github/workflows/release.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
           helm repo update
   
       - name: Create k8s Kind Cluster
-        uses: helm/kind-action@v1
```

Comment:
Note that previous runs of this action do not state which exact version of this action was used. https://github.com/chainguard-dev/auto-deploy-demo/actions/runs/12419382619/job/34674567924#step:12:1